examples/chatgpt/rag-quickstart/gcp/Getting_started_with_bigquery_vector_search_and_openai.ipynb

```diff
@@ -0,0 +1,945 @@
LvdUKlUwsbGRqhWq2/eA5/gRxKgSmjO+Ph4uHTpUnz24cOH4fHjxwmVDhMgQIAAAQIECGQVuHLlSrh9+3Z8mQBVVj31BPItIECV7/6ZPQECBAgQIECAAAECBAgQIECAAAEC+RcQoDrcQwGqwybxkShh1+l04vuNRqP/ERMJ5Q4TIECAAAECBAgQIECAAAECBAgQIGan6U4AABSWSURBVECAAAECBAgQIECAAAECBAgQOPECU1NT4dSpU2F/fz9sbW2d+Pm+GRMUoEpQFqBKgHGYAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAQIEEBKgSmnn27NkQfZxMdFtbWwubm5sJlQ4TIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIJBXAQGqvHbOvAkQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQGFhAgGpgQgMQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIJBXAQGqvHbOvAkQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQGFhAgGpgQgMQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIJBXAQGqvHbOvAkQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQGFhAgGpgQgMQIECAAAECBAhkFajVaqHdbseX1ev10Gq1sg6hngCBnApMT0+Hu3fvxrOfm5sLS0tLOV2JaRMgQIAAAQIECBAgQIAAAQIECBAgQCCfAqurq2F8fDxsb2+Hl19+OZ+LOOZZC1AdM6jhCBAgQIAAAQIEjhYQoDraSAWBogoIUBW1s9ZFgAABAgQIECBAgAABAgQIECBAgEBeBLrdbqhUKmFjYyNUq9W8THuo8xSgSuA9c+ZMeOmll+Kza2trYXNzM6HSYQIECBAgQIAAgawCly9fDjdu3IgvW1hYCNHrLTcCBMohMDExEWZnZ+PF3rlzJ9y/f78cC7dKAgQIECBAgAABAgQIECBAgAABAgQInBABAarDjRCgOmwSH4kSdp1OJ77faDRCs9lMqHSYAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAIE8Czw5mLz//o/BQYDqSe928HmPfP6Pjz3j/zP2gD1gD9gD9oA9YA/YA/aAPXDce+AjH/nIk/X19Scf/ehHU78HO3hr6bguqn3ve9+bWvs93/M9/dp3vOMdqbXvf//7+7VHre0Tn/hEXPvBD34wdcxv+qZv6o/5wz/8w6m173nPe/q1b3/721Nrf/3Xfz2u/YM/+IPUui/90i/tj/nKK6+k1r7zne/s1x581GJq7c/93M/1a7/kS74ktfaP/uiP4tqlpaXUuq/92q/tj/mjP/qjqbXvfve7+7Xf+I3fmFr7oQ99KK5tt9updVHP1w/2VPTf+973vtTa7/iO7+jXfvd3f3dq7fz8fL/2K77iK1Jrf/d3fzeu/c3f/M3Uuq/5mq/pj/njP/7jqbXf//3f36+dnJxMrT14d8i49lOf+lRqXWT16U9/Oq79lV/5ldTab/3Wb+0//sFH+KbW/sRP/ES/9q1vfWtqrecLzxfrr///1fOF5wvPF15fpL1u8/oiPPH6wuuL3t+Zvh/x/YjvR3w/kvZ3pp9fhCd+fuHnF72/M8v684u054gSnfND/6c1e3x8/MnMzEz839ve9rbUH9o97XrH7Ct7wB6wB+wBe8AesAfsAXvAHrAHsu2Bv/qrv4p/j2VnZyf1e7AoPNK7HRXKOfho9l7pk3e9612p40ZhpN7tC7/wC1NrP/e5z8WlUYgkrc8HH1naG/LJUQGmX/qlX+rXXrhwIXXcg48+jGt3d3dT66IfkPduv/3bv51aG4WSercf+qEfSq1ttVq90idf/uVfnlq7t7cX137mM59JrXvhhRf6Y/7CL/xCam0U4Ordvv3bvz21NvrhV3T77Gc/m1r3RV/0RXFd9MfHPvax1NroB2m928FHUqbWfvjDH+6VPnnLW96SWvv3f//3ce329nZq3Td8wzf0x/zABz6QWvvTP/3T/dooJJe2X6NgYHT7z//8z9S6aIz/+q//imtv376dWvud3/mdcV30x0/91E+l1v7qr/5qv/brvu7rUms9X3i+6G0WzxeeLzxfpAekvb7w+sLrC68ven9n+n7E9yO+H/H9SNr3g35+EZ74+YWfX/T+zizrzy/SniNKdC7bD7RLBJP6gzoO9o09YA/YA/aAPWAP2AP2gD1gD9gDx7sHBCIEIno/qBKIEIgQiBCISPs7VuAyPBGIEIjo/Z0pECEQIRAhEJH2d6ZAhECEX+jwCx1+AcwvgPVeN/oFML8AdtQvjKa9pijLuf/3+kIPvrgRIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECgXAICVOXqt9USIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIPAGAQGqN2C4S4AAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIBAuQQEqMrVb6slQIAAAQIECJwIgUqlEi5cuBDP5cGDB2Fvb+9EzMskCBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIEyicgQFW+nlsxAQIECBAgQGDkArVaLbTb7Xge9Xo9tFqtkc/JBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBMopIEBVzr5bNQECBAgQIEBgpAICVCPl9+AECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQJvEBCgegPGG+9evHgxLC4uxoeWl5fDysrKG0+7T4AAAQIECBAgMIDAuXPnwrVr1+IRVldXw9bW1gCjuZQAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIDA8wsIUCXYVavV0Ol04rONR
```

Comment:
maybe change this to openai_api_key instead in case it's not set as env variable, so it's just referencing the variable you defined above